en/docs/administer/monitoring.md

```diff
@@ -3,12 +3,15 @@
 See the following topics for information on the various monitoring
```

Comment:
Please refer to PR #517 as certain changes were introduced to the Monitoring section. Certain changes in this PR may revert those changes.